CHANGELOG.md

```diff
@@ -1,17 +1,35 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
 ## 1.1.0 - 2024-06-25
```

Comment:
Format is important, please use the square brackets around versions (see https://keepachangelog.com/en/1.0.0/ or bclconvert repo for an example)